site/docs/v1/tech/messengers/message-templates.adoc

```diff
@@ -0,0 +1,69 @@
+---
+layout: doc
+title: Message Templates
+description: Configuring a Message Template
+---
+
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Adjust a Message Template
+
+A message template allows you to customize how your messenger communicates with your audience. You can add custom styles and text using FreeMarker.
```

Comment:
"how your messenger communicates with your audience" -> "how the message sent to your audience"